### ISSUE TITLE
Error: 'TypeError: this.activateMode' / lodash / Underscore

### ISSUE THREAD

**Issue by silversmurf** [ahmadawais/create-guten-block#133]
### Expected Behavior

I've created a block with cgb and used it within a page. After that I've inserted the default block **Image**.

### Actual Behavior

Immediately after inserting the **Image** block, it comes up these error:
```
TypeError: this.activateMode is not a function
    at media-views.min.js?ver=5.0.2:1
    at arrayEach (blocks.build.js?ver=1:3683)
    at Function.forEach (blocks.build.js?ver=1:12511)
    at n._createModes (media-views.min.js?ver=5.0.2:1)
    at n.initialize (media-views.min.js?ver=5.0.2:1)
    at n.initialize (media-views.min.js?ver=5.0.2:1)
    at n.initialize (media-views.min.js?ver=5.0.2:1)
    at n.e.View (backbone.min.js?ver=1.2.3:1)
    at n.constructor (wp-backbone.min.js?ver=5.0.2:1)
    at n.constructor (media-views.min.js?ver=5.0.2:1)
```
### Steps to Reproduce

1. `node -v`: v9.11.1
2. `npm -v`: 5.6.0
3. `yarn --version` (if you use Yarn): -
4. `npm ls cgb-scripts` (if you haven’t ejected): -- cgb-scripts@1.11.1
5. Operating system: Win10
6. Browser and version (if relevant): Latest Chrome

### Possible Solution

1. Execute within your plugin directory `npm i --save babel-plugin-lodash`
2. Go to `[your-plugin-directory]/node_modules/cgb-scripts/config`
3. Open file `webpack.config.dev.js`
4. Search for line(s) `presets: [ require.resolve( 'babel-preset-cgb' ) ],`
5. Insert new line: `plugins: [ 'lodash' ],`
6. Open file `webpack.config.prod.js`
6. Repeat steps 4 + 5

After that you can use image plugin again w/o errors.

**Comment by Masterjoa** [comment]
I'm experiencing the same issue when Jetpack is enabled.
But the proposed solution doesn't work :/

**Comment by sharazghouri** [comment]
I did Possible Solution Steps but problem not solved.

**Comment by ahmadawais** [comment]
Should be fixed in the update of cgb-scripts: 1.14.0 => 1.15.0
